lms/static/sass/features/_course-experience.scss

```diff
@@ -406,7 +413,7 @@
 
 #expand-collapse-outline-all-button {
   float: right;
-  background-color: $white;
+  background-color: transparent;
```

Comment:
not a big deal, but from what i remember, "transparent" may not be compatible with all browsers?  i think a sure way to do this is keep the background color white and add an "opacity" attribute.   Or possibly set the background color to #fff00 ?  idk just a thought

Reply:
As far as I can tell with a quick internet search, I think it's well supported in this context?